plugin/adsk/scripts/mayaUsd_preferenceTab.mel

```diff
@@ -434,8 +439,8 @@ global proc mayaUsd_PrefHoldState(string $mode)
 				`optionVar -q mayaUsd_MakePathRelativeToImageEditTargetLayer`
 				usdPrefFileDependenciesRelativeToEditTargetLayer;
 		}
-		if (`checkBoxGrp -exists usdPrefUntexturedMode`) {
-			checkBoxGrp -e -v1
+		if (`radioButtonGrp -exists usdPrefUntexturedMode`) {
+			radioButtonGrp -e -v1
```

Comment:
radioButtonGrp doesn't have -v1 flag, you need to use -sl flag (like above) and convert the boolean option: false (0), true (1) into the select flag which 1-based.